### PR TITLE
Misc Style & Configuration

### PR DESCRIPTION
**Pull request by clairelin135**
### What does this PR do?
1. Replace cat image
It seems like there doesn't exist any current functionality to upload images representing a project. Until this is introduced, I replaced the cat image with a filler image and removed the hover styling that made the background black.

2. Removes yellow text on hover for links
Previously, this hid certain links on the navbar.

3. Added ellipsis to truncate wraparound text
I wanted to maintain the grid/tile format of the browse projects page, so I thought the best solution was to replace wraparound text with an ellipsis rather than expand the tile. Not sure if this is something we want to keep.

4. Added configuration to use AWS S3 for Active Storage
Modified production settings and added the AWS S3 gem. Not 100% sure if this works because I didn't test it myself.

### Status
READY

### Description of Testing Done
Tested frontend fixes.

### Todos
Perform testing on production to ensure that images are correctly stored.

### Screenshots
1. Replace cat image
Default bg color is white, on hover is yellow
image

3. Added ellipsis to truncate wraparound text
image

### CODE REVIEW

File: app/javascript/components/utils/ProjectCard.jsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ class ProjectCard extends React.Component {
   constructor(props) {
     super(props);
     this.goToProject = this.goToProject.bind(this);
+    console.log(props);
```
Comment (ethanlee16) [inline app/javascript/components/utils/ProjectCard.jsx]: Let's clean this up!

**Comment by ethanlee16** [comment]
I'll configure an S3 bucket now and put the keys on our staging deployment to test uploads. If all goes well, I'll give this a stamp!

**Comment by ethanlee16** [comment]
Tricky thing I found out last night - the project is not actually using Active Storage 😞...so the configuration needs to be setup on Paperclip's end. I'll dig more into this if I can tonight, but the IAM role and S3 bucket config is all setup and in Heroku now!

**Comment by clairelin135** [comment]
@FranCogMents resolved the line height truncating issue in my latest commit. @ethanlee16 added config for paperclip! Let me know if it doesn't work.

File: config/environments/production.rb [hunk 2]
```diff
@@ -104,4 +105,14 @@
...
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.paperclip_defaults = {
+    storage: s3,
+    s3_credentials: {
+      bucket: ENV.fetch('S3_BUCKET_NAME'),
+      access_key_id: ENV.fetch('AWS_ACCESS_KEY_ID'),
+      secret_access_key: ENV.fetch('AWS_SECRET_ACCESS_KEY'),
+      s3_region: ENV.FETCH('AWS_REGION'),
```
Comment (ethanlee16) [inline config/environments/production.rb]: I believe bucket here should be “AWS_BUCKET” and region should be “AWS_BUCKET_REGION,” as in the last commit!

**Pull request merged**